packages/reactivity/src/effect.ts

```diff
@@ -188,28 +198,52 @@ export function effect<T = any>(
   return runner
 }
 
+/**
+ * Stops the effect associated with the given runner.
+ *
+ * @param runner Association with the effect to stop tracking.
```

Comment:
I wonder if we should mark anything not documented in the [public API](https://vuejs.org/api/reactivity-core.html#ref) as `@internal`? I know these are exported, I'm not sure if that is for public consumption or for other reasons (like people building tooling).

Since these are not documented publicly, I am not sure if there is a guarantee they won't change (I could be wrong about this, though). Marking them as `internal` might help prevent users from accessing APIs that are not guaranteed to remain consistent between releases.

Reply:
That's a nice touch! I'll update the PR soon-ish :D

Reply:
Update: If I understand correctly, the `@internal` tag is a "[Modifier](https://tsdoc.org/pages/spec/tag_kinds/#modifier-tags)" that are meant for adding to a whole doc block instead of just a `@parameter` line.

Reply:
Isn't `effect` considering internal? I don't see it in the docs.

Reply:
It's internal for the Vue API surface (in which case `watchEffect` is preferred to integrate with rendering behavior), but "public" if using `@vue/reactivity` directly.

Reply:
> It's internal for the Vue API surface (in which case `watchEffect` is preferred to integrate with rendering behavior), but "public" if using `@vue/reactivity` directly.

Yup, that's exactly why I decided to add the comment